### DISCUSSION TITLE
Color change doesn't do anything?

### DISCUSSION THREAD

**Discussion by fantastique2** [tailwindlabs/tailwindcss#3867] [Help]
![image](https://user-images.githubusercontent.com/81512067/112755550-c4646d80-8fe9-11eb-8dbe-b20feb1fc10d.png)
I updated the color from teal to pink through the repo on my github, but the color is still the same on my site preview (tried reloading) (https://github.com/neurochemistrydude/gatsby-starter-resume-cms/blob/master/tailwind.config.js).
Any pointers? Thanks!

**Comment by simonswiss** [comment]
Hey!

This should definitely work. Actually, I cloned your repo and upon running `yarn start`, I got greeted with a pink heading :)

![image](https://user-images.githubusercontent.com/485747/112789279-782c3280-90a8-11eb-8261-12129c44c985.png)

Did you change anything since posting this? Do you still need help?
